botCommands/points.js

```diff
@@ -161,9 +154,23 @@ registerBotCommand(awardPoints.regex, awardPoints.cb);
 const points = {
   regex: /(?<!\S)\/points(?!\S)/,
   async cb({
-    content, client, channel, guild,
+    content, author, client, channel, guild,
   }) {
     const userIds = getUserIdsFromMessage(content, /<@!?(\d+)>/g);
+    if (userIds.length === 0) {
```

Comment:
What if instead of replicating the logic, we instead push the author's ID to `userIds` and let the loop run? That way we have DRY code.

Reply:
Don't know why I didn't think to do it that way, amazing call!